apps/glusterfs/volume_entry_create.go

```diff
@@ -55,7 +55,10 @@ func (v *VolumeEntry) updateMountInfo(db wdb.RODB) error {
 			if err != nil {
 				return err
 			}
-			hosts = append(hosts, node.StorageHostName())
+			if node.isOnline() {
```

Comment:
This part is wrong: A node being offline just means heketi won't place new brick on it. This is typically a temporary maintenance state. It can be a step towards node deletion, but before a node is deleted, we should not remove the node from the mount info! It is still a valid node of the gluster cluster.